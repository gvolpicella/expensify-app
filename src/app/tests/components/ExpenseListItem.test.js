import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';

configure({ adapter: new Adapter() });

test('should render ExpenseListItem with expense', () => {
  const expense = expenses[0];
  const wrapper = shallow(<ExpenseListItem {...expense} />);
  expect(wrapper).toMatchSnapshot();
});