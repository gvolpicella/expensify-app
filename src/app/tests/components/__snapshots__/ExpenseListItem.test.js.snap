// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with expense 1`] = `
<div
  className="expense"
>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      gum
    </h3>
  </Link>
  <p>
    $1.15
    -
    1st January 1970
  </p>
</div>
`;
